src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Making your website "cross-origin isolated" using COOP and COEP
+subhead: >
+  Use COOP and COEP to set up a cross-origin isolated environment and enable
+  powerful features like `SharedArrayBuffer`, `performance.measureMemory()` or
+  the JS Self-Profiling API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Use COOP and COEP to set up such an environment and
+  enable powerful features like `SharedArrayBuffer`,
+  `performance.measureMemory()` or the JS Self-Profiling API.
+authors:
+  - agektmr
+hero: hero.png
+date: 2020-04-13
+updated: 2020-04-16
+tags:
+  - post
+  - security
+---
+Some web APIs increase the risk of side-channel attacks like Spectre. To 
+mitigate that risk, browsers offer an opt-in-based isolated environment called 
+cross-origin isolated. With a cross-origin isolated state, the webpage will be 
+able to use privileged features including:
+
+* [`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+  (required for WebAssembly Threads)
+* [`performance.measureMemory()`](/monitor-total-page-memory-usage/)
+* [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/)
+
+The cross-origin isolated state also prevents modifications of
+`document.domain`. (Being able to alter `document.domain` allows communication
+between same-site documents and has been considered a loophole in the
+same-origin policy.)
+
+{% Aside 'caution' %}
+These powerful features and the prevention of `document.domain` modification 
+are not yet enabled in Chrome as of version 83. We'll update this post as they 
+become available.
+{% endAside %}
+
+To opt in to a cross-origin isolated state, you need to send the following two 
```

Comment:
```suggestion
To opt in to a cross-origin isolated state, you need to send the following
```

Removing `two` doesn't change the meaning of the sentence and prevents the article from looking incorrect if we later add more headers but forgot to update this text.